realm/realm-library/src/main/java/io/realm/internal/OsList.java

```diff
@@ -18,6 +18,7 @@
     private static final long nativeFinalizerPtr = nativeGetFinalizerPtr();
     private final ObserverPairList<CollectionObserverPair> observerPairs =
             new ObserverPairList<CollectionObserverPair>();
+    private int notificationCount = 0; // Keeps track of how many times a notification has been
```

Comment:
doesn't seem to be needed?

Reply:
related to the `called` attribute for debugging?